docs/processing/qaqc-criteria-FC.md

```diff
@@ -0,0 +1,32 @@
+# QA/QC criteria for denoised data
+
+## DOF distribution
+
+- [ ] Check that the effective DOF for each session has a large positive value.
+    If some DOFs are near zero or negative, revise the preprocessing to increase the motion censoring threshold, include fewer regressors, or tweak/eliminate band-pass filtering.
+
+## FC distributions
+
+- [ ] Check that the FC distributions appear similar across sessions and are gaussian, centered approximately around zero with a small positive mean <sup>[1]</sup>.
+    If the FC distributions appear shifted towards positive values, skewed, flat, or bimodal, the denoising was either too aggressive for the number of functional frames available, or the BOLD signal is still contaminated with residual noise.
+    - [ ] If the sessions with a non-conform distribution present a DOF close to zero or negative, revise the preprocessing to increase the motion censoring threshold, include fewer regressors, or tweak/eliminate band-pass filtering.
+    - [ ] If the DOF has a large positive value, exclude the sessions with the non-conform distribution.
+
+## QC-FC distributions
+
+- [ ] Check that the QC-FC distribution<sup>[2]</sup> and the null distribution represented by the red dotted line are approximately equivalent by assessing the QC-FC% value corresponding to the distance between the observed and the null distributions. 
+    - [ ] If the QC-FC distribution does not meet the 95% cutoff, revise the preprocessing by lowering the motion censoring threshold, including more regressors, or tweaking the band-pass filtering.
+    - [ ] If the 95% cutoff is still not met after revising the preprocessing, eliminate all sessions associated with extreme IQM outliers.
+    - [ ] If the 95% cutoff is still not met, iteratively exclude the session with the highest `fd_perc` until that cutoff is met.
```

Comment:
Write action before condition:

```suggestion
    - [ ] Revise the preprocessing by lowering the motion censoring threshold, including more regressors, or tweaking the band-pass filtering if the QC-FC distribution does not meet the 95% cutoff.
    - [ ] Exclude sessions associated with extreme IQM outliers if the 95% cutoff is still not met after revising the preprocessing.
    - [ ] Iteratively exclude the session with the highest `fd_perc` until that cutoff is met if the 95% cutoff is still not met.
```

Reply:
In the first item, what does "tweak the band-pass filter" mean?

is it narrower, wider and what side needs to be tweaked first?